mbed-cryptoauthlib.lib

```diff
@@ -1 +1 @@
-https://github.com/janjongboom/mbed-cryptoauthlib/#c74828b1bbdf67281ad4689b05d88b2b32e86c75
+https://github.com/ARMmbed/mbed-cryptoauthlib/#6c0cacc6bb1428cff8b7e252c86ecc0f03d83ce7
```

Comment:
Style: Should there be a newline at the end of this file? It looks like there used to be one, but we've removed it.